octobot_backtesting/importers/exchanges/exchange_importer.py

```diff
@@ -37,13 +38,10 @@ def __init__(self, config, file_path):
         await self.database.initialize()
 
         # load description
-        description = (await self.database.select(DataTables.DESCRIPTION, size=1))[0]
-
-        version = description[1]
-        if version == "1.0":
```

Comment:
Why removing description versionning support ?

Reply:
It's now in get_database_description:
![image](https://user-images.githubusercontent.com/9078616/72665250-bccd3000-3a06-11ea-8ca1-b447653e7822.png)